src/Hedgehog/Range.fs

```diff
@@ -93,6 +94,7 @@ module Range =
     //
 
     [<AutoOpen>]
+    [<System.ComponentModel.EditorBrowsable(System.ComponentModel.EditorBrowsableState.Never)>]
```

Comment:
ref: https://stackoverflow.com/questions/27253613/why-editorbrowsablestate-never-exists

Reply:
It'd be nice to be able to discover those via IntelliSense internally, when working in this codebase. Is `Never` going to prevent that also?

Reply:
Yes, it prevents you from seeing it even when you're in the same fsproj.

FWIW we only have one occurrence of `Range.Internal` since its `[<AutoOpen>]`

Reply:
Is it practical, though? Perhaps we can just take it out of the `Range.Internal` namespace and mark it as `internal`. Can be done separately, of course.

Reply:
Unfortunately, `inline` functions can't have `internal` scope if they're used in publicly scoped functions.

I'm not sure why they're `inline` to begin with, honestly.

Reply:
I can't remember why I did it that way, but it's a good tradeoff to have it as `internal` and get rid of the namespace and the inline part, which my gut feeling says it won't hurt the performance.

Reply:
`inline` is required to work with SRTPs - I elaborated on the comment on the `Internal` module in a new commit.